clouddrift/adapters/gdp1h.py

```diff
@@ -510,6 +510,9 @@ def preprocess(index: int, **kwargs) -> xr.Dataset:
     # rename variables
     ds = ds.rename_vars({"longitude": "lon", "latitude": "lat"})
 
+    # Cast float64 variables to float32 to reduce memory footprint.
+    ds = gdp.cast_float64_variables_to_float32(ds)
```

Comment:
Is this where you should add variables_to_skip = ["lon", "lat", "time"]?`

Reply:
That's the default, see the function definition.